aldryn_translation_tools/admin.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.conf import settings
+
+from cms.utils.i18n import get_current_language
+from cms.utils.urlutils import admin_reverse
+
+
+class AllTranslationsMixin(object):
+    class Media:
+        css = {'all': ('css/admin/all-translations-mixin.css', ), }
+
+    def all_translations(self, obj):
+        """
+        Adds a property to the list_display that lists all translations with
+        links directly to their change forms. Includes CSS to style the links to
+        looks like tags with color indicating current language, active and
+        inactive translations.
+
+        A similar capability is in HVAD, and now there is this for
+        Parler-based projects.
+        """
+        available = list(obj.get_available_languages())
+        current = get_current_language()
+        langs = []
+        for code, title in settings.LANGUAGES:
+            classes = ["lang-code", ]
+            if code == current:
+                classes += ["current", ]
+            if code in available:
+                classes += ["active", ]
+                title += " (translated)"
```

Comment:
This is really confusing - are you changing the value in the settings module, or creating a new string? I think the latter, but then it would be a bit easier to read if you used a new variable.